dags/homelessness/sync_rap_data.py

```diff
@@ -0,0 +1,95 @@
+"""
+Pull data from RAP FTP and sync with ArcGIS Online
+"""
+import pandas as pd
+import ftplib
+import arcgis
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.base_hook import BaseHook
+from datetime import datetime, timedelta
+
+def get_file(ftphost, ftpuser, ftppassword, filename):
+    ftp = ftplib.FTP(host = ftphost)
+    ftp.login(user = ftpuser, passwd = ftppassword)
+    ftp.cwd("Upload")
+    ftp.retrbinary('RETR %s' % filename, open("tmp/%s" % filename, 'wb').write)
```

Comment:
I think this should be `"/tmp/"` (for some context, current file permissions are a bit messed up, and this should allow the system to clean up after you).